cmd/rpc.go

```diff
@@ -175,3 +191,60 @@ func startNativeSpaceBridgeRpcServer(ctx context.Context, wg *sync.WaitGroup) {
 	server := rpc.MustNewNativeSpaceBridgeServer(&config)
 	go server.MustServeGraceful(ctx, wg, config.Endpoint, rpcutil.ProtocolHttp)
 }
+
+// startDebugSpaceRpcServer starts RPC server for geth debug API
+func startDebugSpaceRpcServer(ctx context.Context, wg *sync.WaitGroup) {
+	httpEndpoint := viper.GetString("debugrpc.endpoint")
+	ethNodeRPCURL := viper.GetString("node.router.ethnoderpcurl")
+	logrus.Debug("Debug Space RPC server HTTP endpoint=", httpEndpoint, ", ethNodeRPCURL=", ethNodeRPCURL)
+	client, err := gethrpc.DialHTTP(ethNodeRPCURL)
+	if err != nil {
+		logrus.WithError(err).Fatal("Failed to create rpc client")
```

Comment:
we should not `Fatal` here.

`Fatal` will not run `defer` whereas `panic` will: https://quasilyte.dev/blog/post/log-fatal-vs-log-panic/

We do have a `defer` outside `https://github.com/scroll-tech/rpc-gateway/blob/feat/add_debug_rpc/cmd/root.go#L68`: https://github.com/scroll-tech/rpc-gateway/blob/feat/add_debug_rpc/cmd/root.go#L68

Reply:
also, what about we add more detail for this error message: `Failed to create rpc client for debugSpaceRpcServer`?

Reply:
`Failed to create rpc client for debugSpaceRpcServer` error message added.

Reply:
changed `Fatal` to `Panic`